src/confluence/subcommands/merge.py

```diff
@@ -183,17 +181,41 @@ def main(args):
     setup_logging(args.loglevel)
     _logger.debug(f"Starting merge operation...")
     # read input files
-    _logger.debug(f"Reading files: {args.filelist}")
+    _logger.debug(f"Validating files: {args.filelist}")
+    try:
+        validate_files(args.filelist)
+    except ValueError:
+        sys.exit(0)
```

Comment:
This always validates the file list, even when the user doesn't request it. I'm still not sure what would be the value of validating the file list. We had talked about validating the files that are found in the DataFrame. ("FILE: ...." column.)